testing/node/gsub/issue-113/gsub.test.js

```diff
@@ -0,0 +1,77 @@
+import { Font } from "../../../../lib-font.js";
+
+const font = new Font("GSUB type 7 redirect testing");
+
+describe("GSUB type 7 checks", () => {
+  beforeAll(async (done) => {
+    font.onerror = (err) => {
+      throw err;
+    };
+    font.onload = async () => done();
+    font.src = `./fonts/Lato-Regular.ttf`;
+  });
+
+  test("font loaded", () => {
+    expect(font.opentype).toBeDefined();
+  });
+
+  test("has all expected type 7 redirects", () => {
+    const sequence = [];
+    const { GSUB } = font.opentype.tables;
+
+    // run through all scripts
+    GSUB.getSupportedScripts().forEach((script) => {
+      // and all languages for each script
+      GSUB.getSupportedLangSys(script).forEach((lang) => {
+        let langSysTable = GSUB.getLangSysTable(script, lang);
+        // and all features for each langauge
+        GSUB.getFeatures(langSysTable).forEach((feature) => {
+          // and all lookups for each feature
+          feature.lookupListIndices.forEach((id) => {
+            const lookup = GSUB.getLookup(id);
+            const type = lookup.lookupType;
+            const cnt = lookup.subTableCount;
+            // and all subtables for each lookup
+            for (let i = 0; i < cnt; i++) {
+              const { type: sType, format } = lookup.getSubTable(i);
+              sequence.push([id, type, lang, i, sType, format]);
+            }
+          });
+        });
+      });
+    });
+
+    expect(sequence.slice(0,30)).toEqual([
+      [26, 7, "dflt", 0, 6, 3],
```

Comment:
I'd include the full list if it wasn't 200+ lines long. This feels representative enough =)